account_invoice_report_grouped_by_picking/models/account_move.py

```diff
@@ -48,7 +48,7 @@ def lines_grouped_by_picking(self):
                         qty = so_line.product_uom_qty
                         picking_dict[key] += qty
                         remaining_qty -= qty
-            if not float_is_zero(
+            if line.product_id and not float_is_zero(
                 remaining_qty, precision_rounding=line.product_id.uom_id.rounding
```

Comment:
```suggestion
            if not float_is_zero(
                remaining_qty, precision_rounding=line.product_id.uom_id.rounding or 0
```

Reply:
`line.product_id.uom_id.rounding` this already returns 0. The function checks that the value is greater than 0 and since it is not, the error occurs.

Reply:
Rectified my suggestion then

Reply:
Change applied